variables.tf

```
@@ -120,3 +126,21 @@ variable "provisioned_concurrency" {
 
   default = null
 }
+
+variable "schedule" {
+  description = "Make the lambda run on a schedule. For example, cron(0 20 * * ? *) or rate(5 minutes)"
+  type        = string
+  default     = null
+}
+
+variable "cloudwatch_logs_retention_in_days" {
```

Her ville jeg heller kalt det:
```suggestion
variable "logs_retention_in_days" {
```

Implisit at det er cloudwatch her.

---

done